tests/main/views/test_requirement_task_list.py

```diff
@@ -8,323 +8,478 @@
 from ...helpers import BaseApplicationTest
 
 
-class TestBriefSummaryPage(BaseApplicationTest):
+class RequirementsTaskListPageTest:
```

Comment:
Tiny thing but can we call this class `TestRequirementsTaskListPage` instead please?

Reply:
We can't because then pytest would run it, and it's a base class, not a test suite on it's own. We could call it `BaseRequirementsTaskListPageTest` to make that clear?

Reply:
(Or if it's a shared test class, let's call it that.)

Reply:
Done